src/SourceBuild/content/binary-tooling.sh

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env bash
+
+### Usage: $0
+###
+###   Runs the binary tooling as needed for source-build scenarios. The tooling is responsible for
+###
+### Options:
+###   --binaries-keep-file <FILE>     Path to the file containing the list of allowed binaries to keep. Default is
+###                                   src/installer/src/VirtualMonoRepo/allowed-binaries.txt
+###   --binaries-remove-file <FILE>   Path to the file containing the list of allowed binaries to remove.
```

Comment:
What is the use case for this?

Reply:
I updated the description (https://github.com/dotnet/installer/pull/18726/files#diff-65ef9ccf3668bfa54f7b0dbee3732b2aa8b7cb5868580d8085102ad13078f1b9R24) so that the purpose of this file is more clear. It is intended to be the "baseline" for the binaries that are allowed in the VMR that we disallow for source-build. Things that will go in this are cloaks being removed in https://github.com/dotnet/installer/pull/18563 for example